APMrover2/radio.cpp

```diff
@@ -148,3 +154,33 @@ bool Rover::trim_radio()
 
     return true;
 }
+
+int16_t Rover::get_throttle_control_in(bool no_deadzone) const
```

Comment:
I know this was taken from Plane.  But why is this in the `Rover::` namespace not `RC_Channel_Rover::`.  So long as it only affect RC input (and RC overrides) I don't think it should be leaving the `RC` namespace...